models/org_team.go

```diff
@@ -517,10 +517,14 @@ func AddTeamMember(team *organization.Team, userID int64) error {
 		}
 	}
 
+	if err := committer.Commit(); err != nil {
+		return err
+	}
+
 	// watch could be failed, so run it in a goroutine
```

Comment:
Is the go routine still necessary, if we now commit before that?

Reply:
I think we can keep it still running in go routine because it may takes much time if one org has many repositories.  And in future we can batch update them before we move it back from go routine.